cluster/gce/util.sh

```diff
@@ -3833,7 +3862,8 @@ function test-teardown() {
     "${NODE_TAG}-http-alt" \
     "${NODE_TAG}-nodeports"
   if [[ ${MULTIZONE:-} == "true" && -n ${E2E_ZONES:-} ]]; then
-    local zones=( ${E2E_ZONES} )
+    local zones
+    while IFS= read -r line; do zones+=("$line"); done <<< "${E2E_ZONES}"
```

Comment:
are you sure this is the same as before? I feel like this new logic expects E2E_ZONES to be multi-line, and I don't think that's correct

Reply:
Yes, you are right. I will update this at once.